src/backend/function/Fusion.Resources.Functions/Fusion.Resources.Functions.csproj

```diff
@@ -1,19 +1,19 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+	  <TargetFramework>net6.0</TargetFramework>
+	  <AzureFunctionsVersion>v4</AzureFunctionsVersion>
```

Comment:
When upgrading function you probably have to do some manualsteps in the function.template.json file

you need to set net-framework-version to v6.0
you need to set FUNCTIONS_EXTENSION_VERSION to ~4 

You might even need to manually update the settings after deploy.


Read more here: https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions